.github/README.md

```diff
@@ -0,0 +1,137 @@
+Chipyard Continuous Integration (CI)
+===========
+
+Website: https://gihub.com/gh/ucb-bar/chipyard/actions
+
+GitHub Actions Brief Explanation
+---------------------------
+
+CI is executed by Github Actions (GA). GA is controlled by `.yml` files in the `.github/workflows/` directory. 
+In our case we have just one workflow named `chipyard-rocket-run-tests.yml`.
```

Comment:
Can we rename this `.yml` file to `chipyard-run-tests.yml` or something similar (we don't exclusively test Rocket).

Reply:
Done